Sources/ApolloCodegenTestSupport/MockApolloCodegenConfiguration.swift

```diff
@@ -22,4 +22,16 @@ extension ApolloCodegenConfiguration {
       apqs: apqs
     )
   }
+
+  public static func mock(
```

Comment:
I added something similar in my most recent PR, so we should look and combine these after merging everything in.